packages/clients/test/aptos/integration.env.ts

```diff
@@ -3,19 +3,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Aptos, AptosConfig, Network } from "@aptos-labs/ts-sdk";
 import {
   GasStationClient,
   KeyClient,
   WalletClient,
+  createAptosClient as createAptosClientImpl,
```

Comment:
Why does this need to be `as createAptosClientImpl`?

Reply:
I was originally playing around with some stuff, this was the detritus of that. I'll adjust